gluten-core/src/main/scala/io/glutenproject/execution/ExpandExecTransformer.scala

```diff
@@ -223,6 +226,13 @@ case class ExpandExecTransformer(projections: Seq[Seq[Expression]],
       return false
     }
 
+    // Fallback if projections have scalar function, which is not supported currently.
+    if (projections.exists(
+          _.exists(expr => !expr.isInstanceOf[Attribute] && !expr.isInstanceOf[Literal]))) {
+      logWarning("There are scalar functions in expand node, which is not supported currently.")
+      return false
```

Comment:
@JkSelf Do we have such validation in our native validator for Expand?

Reply:
Looks Velox backend does not need this, is that right?

Reply:
> Looks Velox backend does not need this, is that right?

ok, have made the check only in CH backend